### DISCUSSION TITLE
How to override a miss

### DISCUSSION THREAD

**Discussion by Reyororo** [FunkinCrew/Funkin#1878] [Q&A]
I want to set it so that pressing a key when my boyfriend sings, like in kade engine, doesn't result in a miss.
Does anyone know how to do this?

**Comment by isakube** [comment]
use kade engine :bruh:

actually don't, use psych

**Reply by Reyororo** [reply]
I would like to know how to add a program that doesn't make mistakes from the ninjamuffin source code, is it impossible?
